package.json

```diff
@@ -640,7 +641,7 @@
     "test": "npm run clean && npm run compile && node ./node_modules/vscode/bin/test",
     "update-deps": "node_modules/.bin/ncu --upgrade --loglevel verbose --packageFile package.json && npm update",
     "coverage:upload": "codecov -f coverage/coverage-final.json",
-    "build": "npm run clean && tslint -p tslint.json && npm run compile"
+    "build": "npm run clean && tslint -p tslint.json && npm run compile && npm run verify"
```

Comment:
Let's run ```npm run verify``` in test script instead so it would be called only in PR check builds.